docs/rules/prefer-reflect.md

```diff
@@ -54,12 +54,12 @@ Examples of **correct** code for this rule when used without exceptions:
 ```js
 /*eslint prefer-reflect: "error"*/
 
-Reflect.apply(undefined, args);
-Reflect.apply(null, args);
+Reflect.apply(obj.fooBar, args);
```

Comment:
This and the following line seem to be examples of the exact same use case. Also, doesn't `Reflect.apply` take three arguments?

It looks to me like maybe the intended examples here were:

```js
Reflect.apply(obj.foo, undefined, args);
Reflect.apply(obj.foo, null, args);
```
and
```js
Reflect.apply(obj.foo, undefined, [arg]);
Reflect.apply(obj.foo, null, [arg]);
```